File: Unidad 5/Java/U5programa8.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 */

package com.mycompany.u5programa8;
import java.util.Random;
/**
 *
 * @author DELL
 */
public class U5programa8 {

    public static void main(String[] args) {
         dado();
    }
    public static void dado(){
        Random aleatorio = new Random();
        int G = aleatorio.nextInt(6);
        System.out.println("Su numero es: " + G);
    }
}

File: Unidad 5/C#/U5programa8.cs
using System;

namespace U5programa8 // Note: actual namespace depends on the project name.
{
    internal class Program
    {
        static void Main(string[] args)
        {
            dado();
        }
        public static void dado()
        {
            Random aleatorio = new Random();
            int G = aleatorio.Next(6);
            Console.WriteLine("Su numero es: " + G);
        }
    }
}
